src/Sylius/Bundle/UserBundle/Controller/UserController.php

```diff
@@ -123,7 +123,7 @@ protected function prepareResetPasswordRequest(Request $request, TokenProviderIn
                 return $this->viewHandler->handle($configuration, View::create($user, 204));
             }
 
-            $this->addFlash('success', 'sylius.user.reset_password.requested');
+            $this->addFlash('success', 'sylius.user.reset_password.requested', ['%userEmail%' => $user->getEmail()]);
```

Comment:
Check to see if the User is not null first before trying to get the email otherwise it will be fatal.

Reply:
ok

Reply:
6 year old comment 🤣